etc/kayobe/seed.yml

```diff
@@ -137,6 +137,30 @@ seed_squid_container:
       - /srv/docker/squid/squid.conf:/etc/squid/squid.conf
       - /var/log/squid:/var/log/squid
 
+# Seed container running a cron service.
+seed_cron_container_enabled: true
+
+seed_cron_container:
+  cron:
+    container_name: cron
+    docker_registry: "{{ lookup('vars', admin_oc_net_name ~ '_ips')[groups.seed.0] }}:{{ pulp_port }}"
+    image: "{{ docker_registry ~ '/' if docker_registry else '' }}stackhpc/{{ kolla_base_distro }}-source-cron"
+    registry_username: "{{ pulp_username }}"
+    registry_password: "{{ pulp_password }}"
+    pre: "{{ kayobe_config_path }}/containers/cron/pre.yml"
+    tag: "yoga-20230220T181235"
```

Comment:
This tag is the main part that concerns me.

1. It won't necessarily be the same for each distro
2. It's in a different place in the config from the other kolla tags, so could easily get out of sync

Reply:
Yes that was my concern as well, but as kolla is not used during this action I couldn't pull it from kolla configs.
Would just putting a filter for specific kolla globals.yml be approperiate?

Reply:
I don't think that could work currently. globals.yml isn't necessarily valid YAML, or even templated to valid YAML

Reply:
I guess for now we could document that tag should be changed during the deployment in that case.